keyboards/dz60/keymaps/coppertop/keymap.c

```diff
@@ -0,0 +1,76 @@
+#include QMK_KEYBOARD_H
+
+//Layers
+#define _QWERTY 0
+#define _FN 	1
+#define _MEDIA 	2
```

Comment:
We highly recommend using an enum rather than defines for the layer names.

```suggestion
enum layer_names {
    _QWERTY,
    _FN,
    _MEDIA,
};